SUMMARY.md

```diff
@@ -299,49 +323,40 @@
     - [Interface and Registers](./hw/ip/usbdev/data/usbdev.hjson)
     - [Device Interface Functions](./sw/device/lib/dif/dif_usbdev.h)
     - [Checklist](./hw/ip/usbdev/doc/checklist.md)
-  - [lowRISC Hardware Primitives](./hw/ip/prim/README.md)
-    - [Two Input Clock](./hw/ip/prim/doc/prim_clock_gp_mux2.md)
-    - [Flash Wrapper](./hw/ip/prim/doc/prim_flash.md)
-    - [Keccak Permutation](./hw/ip/prim/doc/prim_keccak.md)
-    - [Linear Feedback Shift Register](./hw/ip/prim/doc/prim_lfsr.md)
-    - [Packer](./hw/ip/prim/doc/prim_packer.md)
-    - [Packer FIFO](./hw/ip/prim/doc/prim_packer_fifo.md)
-    - [Present Scrambler](./hw/ip/prim/doc/prim_present.md)
-    - [Prince Scrambler](./hw/ip/prim/doc/prim_prince.md)
-    - [SRAM Scrambler](./hw/ip/prim/doc/prim_ram_1p_scr.md)
-    - [Pseudo Random Number Generator](./hw/ip/prim/doc/prim_xoshiro256pp.md)
 
-- [Common SystemVerilog and UVM Components](./hw/dv/sv/README.md)
-  - [ALERT_ESC Agent](./hw/dv/sv/alert_esc_agent/README.md)
-  - [Bus Params Package](./hw/dv/sv/bus_params_pkg/README.md)
-  - [Comportable IP Testbench Architecture](./hw/dv/sv/cip_lib/README.md)
-  - [Common Interfaces](./hw/dv/sv/common_ifs/README.md)
-  - [CSR Utils](./hw/dv/sv/csr_utils/README.md)
-  - [CSRNG Agent](./hw/dv/sv/csrng_agent/README.md)
-  - [DV Library Classes](./hw/dv/sv/dv_lib/README.md)
-  - [DV Utils](./hw/dv/sv/dv_utils/README.md)
-  - [FLASH_PHY_PRIM Agent](./hw/dv/sv/flash_phy_prim_agent/README.md)
-  - [I2C Agent](./hw/dv/sv/i2c_agent/README.md)
-  - [JTAG Agent](./hw/dv/sv/jtag_agent/README.md)
-  - [JTAG DMI Agent](./hw/dv/sv/jtag_dmi_agent/README.md)
-  - [JTAG RISCV Agent](./hw/dv/sv/jtag_riscv_agent/README.md)
-  - [KEY_SIDELOAD Agent](./hw/dv/sv/key_sideload_agent/README.md)
-  - [KMAC_APP Agent](./hw/dv/sv/kmac_app_agent/README.md)
-  - [Memory Backdoor Scoreboard](./hw/dv/sv/mem_bkdr_scb/README.md)
-  - [Memory Backdoor Utility](./hw/dv/sv/mem_bkdr_util/README.md)
-  - [Memory Model](./hw/dv/sv/mem_model/README.md)
-  - [PATTGEN Agent](./hw/dv/sv/pattgen_agent/README.md)
-  - [PUSH_PULL Agent](./hw/dv/sv/push_pull_agent/README.md)
-  - [PWM Monitor](./hw/dv/sv/pwm_monitor/README.md)
-  - [RNG Agent](./hw/dv/sv/rng_agent/README.md)
-  - [Scoreboard](./hw/dv/sv/scoreboard/README.md)
-  - [Simulation SRAM](./hw/dv/sv/sim_sram/README.md)
-  - [SPI Agent](./hw/dv/sv/spi_agent/README.md)
-  - [String Utils](./hw/dv/sv/str_utils/README.md)
-  - [Test Vectors](./hw/dv/sv/test_vectors/README.md)
-  - [Tile Link Agent](./hw/dv/sv/tl_agent/README.md)
-  - [UART Agent](./hw/dv/sv/uart_agent/README.md)
-  - [USB20 Agent](./hw/dv/sv/usb20_agent/README.md)
+- [UVM/SystemVerilog](./hw/dv/sv/README.md)
```

Comment:
Maybe change this to SystemVerilog/UVM to make it consistent with the title used later.